### ISSUE TITLE
Paper-Doubts

### ISSUE THREAD

**Issue by manojsai2** [ChangyuChen347/semi-offline-RL#4]
Hello sir ,
I need some clarifications in the paper can you  please share with us 
1.how reward is calculated ?
2.why 3 rouges scores are calculated?
3.can you suggest any ideas that can still improve this paper further?
4.How fine-tuning is done  and what is the use of it?
5.In Code where the fine-tuning is done?
Please share the following information 
Thank you.

**Comment by ChangyuChen347** [comment]
1. See the code below, the code for calculating ROUGE is the same as normal. https://github.com/ChangyuChen347/semi-offline-RL/blob/9411e261d9ce2a9929da34bc29ec045d225cbfdd/rl_env.py#L206
2. This is following the setup of the previous paper, such as BRIO.
3.  Maybe some overfitting/reward-hacking or generalization patterns associated with this method. The non-autoregressive sampling approach we adopted is different from conventional ones, and the occurrence of overfitting patterns may also be different. Currently, there are no related experiments in the paper.
4. See the loss we used. Finetuning is equivalent to setting the weight of rl loss to zero. Fine-tuning enables the pre-trained model to generate sentences for downstream tasks, like summarization. If you run the RL loss directly without fine-tuning, the performance will be relatively worse.
5. You can set the rl_weight to 0 during the training process, which means you are doing fine-tuning.

**Comment by manojsai2** [comment]
Thank you so much also can tell about policy gradient method in code where it is and how it is usefull??

**Comment by ChangyuChen347** [comment]
> Thank you so much also can tell about policy gradient method in code where it is and how it is usefull??
https://github.com/ChangyuChen347/semi-offline-RL/blob/9411e261d9ce2a9929da34bc29ec045d225cbfdd/rl_env.py#L308 
It's for reward optimization.

**Comment by manojsai2** [comment]
Thank you so much
